deploy/helm/sumologic/README.md

```diff
@@ -303,8 +303,8 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `telegraf-operator.data`                                                  | Telegraf sidecar configuration.                                                                                                                                                                                                                                                                                                                                      | See [values.yaml]                                                                                                 |
 | `opentelemetry-operator.enabled`                                          | Flag to control deploying OpenTelemetry Operator Helm sub-chart.                                                                                                                                                                                                                                                                                                     | `false`                                                                                                           |
-| `opentelemetry-operator.createDefaultInstrumentation`                     | Flag to control creation of default Instrumentation object                                                                                                                                                                                                                                                                                                           | `true`                                                                                                            |
-| `opentelemetry-operator.manager.env.WATCH_NAMESPACE`                      | Used to set value for `WATCH_NAMESPACE` environment variable which specifies Namespace to watch and create Instrumentation objects.                                                                                                                                                                                                                                  | `Nil`                                                                                                             |
+| `opentelemetry-operator.createDefaultInstrumentation`                     | Flag to control creation of default Instrumentation object                                                                                                                                                                                                                                                                                                           | `false`                                                                                                           |
```

Comment:
Isn't this kind of a breaking change? :sweat_smile:

Reply:
Yeah, that seems like a breaking change to me to.

Reply:
Could be a breaking change, but we were treating OT Operator for Tracing as `BETA`. Appropriate comments in the next PR with changelog and release description will be added about this change.